workers.tf

```diff
@@ -141,6 +141,12 @@ resource "aws_iam_role_policy_attachment" "workers_AmazonEC2ContainerRegistryRea
   role       = "${aws_iam_role.workers.name}"
 }
 
+resource "aws_iam_role_policy_attachment" "workers_additional_policies" {
+  count      = "${var.workers_addtional_policies_count}"
+  role       = "${aws_iam_role.workers.name}"
+  policy_arn = "${var.workers_addtional_policies[count.index]}"
```

Comment:
typo: `workers_addtional_policies` instead of `workers_additional_policies`

Reply:
fixed